src/components/skills-quiz/tests/SearchCourseCard.test.jsx

```diff
@@ -132,25 +128,30 @@ const SearchCourseCardWithContext = ({
 
 describe('<SearchCourseCard />', () => {
   test('renders the correct data', async () => {
-    let containerDOM = {};
-    await act(async () => {
-      const { container } = renderWithRouter(
-        <SearchCourseCardWithContext
-          index={testIndex}
-        />,
-      );
-      containerDOM = container;
-    });
+    const { container, history } = renderWithRouter(
+      <SearchCourseCardWithContext
+        index={testIndex}
+      />,
+    );
+
+    const searchCourseCard = await screen.findByTestId('skills-quiz-course-card');
+    expect(searchCourseCard).toBeInTheDocument();
 
     expect(screen.getByText(TEST_TITLE)).toBeInTheDocument();
     expect(screen.getByAltText(TEST_PARTNER.name)).toBeInTheDocument();
+    expect(screen.getByText(TEST_PARTNER.name));
```

Comment:
[curious] Should we be checking if `TEST_PARTNER.name` appears in the document as opposed to its existence on the screen?

`expect(screen.getByText(TEST_PARTNER.name));`
vs
`expect(screen.getByText(TEST_PARTNER.name)).toBeInTheDocument();`

Reply:
Yes :) Fixed.